etc/di.xml

```diff
@@ -11,6 +11,7 @@
  */
 -->
 <config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:ObjectManager/etc/config.xsd">
+    <preference for="Ebizmarts\MailChimp\Helper\Data" type="Ebizmarts\MailChimp\Helper\Data\Proxy" />
```

Comment:
please change this definition for 
`    <type name="Ebizmarts\MailChimp\Block\Mailchimpjs">
        <arguments>
            <argument name="helper" xsi:type="object">Ebizmarts\MailChimp\Helper\Data\Proxy</argument>
        </arguments>
    </type>
`

this way the change affect only to the Mailchimpjs block

Reply:
I did it globally, because I saw usage of the helper in other MailChimp blocks. I think it could also impact performance there (for example on Success page).
But I did not measure performance impact and I didn't get into details of blocks logic.
Let's keep it as you suggested.